File: aparcamiento/AMarotoChivite-aparcamiento/AMarotoChivite-aparcamiento-Java/src/main/java/simulacionParking/enums/TipoVehiculo.java
package simulacionParking.enums;

public enum TipoVehiculo {
    // Tamaño Grande
    CAMION("N"),

    // Tamaño Mediano
    COCHE("C"),

    // Tamaño Pequeño
    MOTO("M"),
    BICI("B"),
    PATINETE("P"),
    VACIO("-");
    private final String t;

    TipoVehiculo(String t) {
        this.t = t;
    }

    public String get() {
        return t;
    }
}

File: aparcamiento/AMarotoChivite-aparcamiento/AMarotoChivite-aparcamiento-Kotlin/src/main/kotlin/simulacionParking/enums/TipoVehiculo.kt
package simulacionParking.enums


enum class TipoVehiculo(private val t: String) {
    // Tamaño Grande
    CAMION("N"),

    // Tamaño Mediano
    COCHE("C"),

    // Tamaño Pequeño
    MOTO("M"),
    BICI("B"),
    PATINETE("P"),
    VACIO("-");

    fun get(): String {
        return t
    }
}